sections/04_study.md

```diff
@@ -266,8 +266,40 @@ application in this area.*
 
 ### Micro-RNA binding
 
-*miRNAs are important biologically, but have neural networks produced anything
-particularly notable in this area?*
+Prediction of microRNAs (miRNAs) in the genome as well as miRNA targets is of
+great interest, as they are critical components of gene regulatory networks and
+are often conserved across great evolutionary distance [@tag:Bracken2016_mirna
+@tag:Berezikov2011_mirna]. While many machine learning algorithms have been
+applied to solve these prediction tasks, they currently require extensive
+feature selection and optimization. For instance, the widely adopted state of
```

Comment:
I don't have any evidence to the contrary, but do you have a sense of how accepted it is that TargetScan is the state of the art?  There are other competitors, but I do tend to see TargetScan used frequently.

I'm wondering whether we need to hedge slightly and say TargetScan is an example of state of the art versus it is *the* state of the art.  If it is clearly the dominant existing method, I prefer to keep the text as-is.